assets/images/logos/README.md

```diff
@@ -4,7 +4,8 @@ The home and /radiuss pages display a logo next to each repo when they appear un
 
 If a repo has its own logo, that should display. If not, then its organization's avatar should display. Every repo should have one logo or avatar; no repo should be "empty."
 
-Logo files should be uploaded to a specific protected OneDrive folder called [Repo logos (all) - standard file names](https://doellnl-my.sharepoint.com/personal/auten1_llnl_gov/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fauten1%5Fllnl%5Fgov%2FDocuments%2FOS%20portal%20%28software%2Ellnl%2Egov%29%20design%20mockups%2FRepo%20logos%20%28all%29%20%2D%20standard%20file%20names).
-- Each file should contain the repo name plus `repo-logo` separated by hyphens (e.g., `MFEM-repo-logo`).
-- Files should be .jpg, .gif, or .png.
-- Contact @hauten if you are unable to access.
+Logo files should follow a naming convention and be added to the directory. All logos must be in .png format.
+- Repository logo: `<repo_name>.png`
+- Directory: `/assets/images/logos/<org name>`
+
+Organization logos are also supported -- `<org_name>.png` in the same directory -- but this site does pull organization avatars from GitHub.
```

Comment:
The wording around organization logos is somewhat unclear. 

What if the Spack org and spack repo have different logos? Should both be placed in the /assets/images/logos/spack/ directory?

Reply:
Maybe should just say that org logos are pulled from GH to avoid confusion.